docs/operation-guide-for-integration-with-each-optee-release.md

```diff
@@ -0,0 +1,112 @@
+
+**Location:** https://nightlies.apache.org/teaclave/teaclave-trustzone-sdk/
+
+For each OP-TEE release, we maintain a comprehensive set of images across multiple 
+architectures (aarch64, x86_64). These images serve dual purposes:
+1. **CI Infrastructure:** Enable automated testing of TAs on QEMUv8 platforms
+2. **Developer Environment:** Provide ready-to-use emulation environments for quick 
+   TA development and testing
+
+#### File Structure
+
+For each OP-TEE release (e.g., 4.8.0), the following files are generated:
+- `aarch64-optee-4.8.0-qemuv8-ubuntu-24.04-expand-ta-memory.tar.gz`
+- `aarch64-optee-4.8.0-qemuv8-ubuntu-24.04.tar.gz`
+- `x86_64-optee-4.8.0-qemuv8-ubuntu-24.04-expand-ta-memory.tar.gz`
+- `x86_64-optee-4.8.0-qemuv8-ubuntu-24.04.tar.gz`
+
+#### Image Types
+
+- **Standard images**: By default, our examples use standard images, which are built 
+  on the OP-TEE repo without changes.
+- **Expand-ta-memory images**: Used by TAs that need large memory, such as TLS examples. 
+  The patch is applied based on the OP-TEE repo codebase. See 
+  [Expanding TA Secure Memory on QEMUv8](expanding-ta-secure-memory-on-qemuv8.md) 
+  for details.
+
+#### Upload Process
+
+1. **Download from Action artifacts:**
+
+   Download all four artifacts from the CI Actions page.
+   Note: If downloading from GitHub Actions, the file will be in zip format. You should 
+   run `unzip` to extract the `*.tar.gz` files.
+
+2. **Upload to nightlies:**
+   
+   You need an Apache account. See [Becoming a Member](https://teaclave.apache.org/becoming-a-member) 
+   for more information.
+
+   Upload each of the four artifacts using curl. Example command:
+   ```bash
+   curl -u YOUR_ASF_ID \
+        -T ./aarch64-optee-4.8.0-qemuv8-ubuntu-24.04.tar.gz \
+        "https://nightlies.apache.org/teaclave/teaclave-trustzone-sdk/"
+   ```
+
+### 3. PR to Bump optee-version.txt
+
+Update the OP-TEE version to enable the environment to download the correct OP-TEE 
+image version.
+
+1. Update the version in: 
+   https://github.com/apache/teaclave-trustzone-sdk/blob/main/optee-version.txt
+2. Create a PR with the version bump
+3. Merge the PR after CI passes
+
+### 4. Build and Publish Dev Docker
+
+After step 3 PR is merged, rebuild the development Docker image.
+
+1. Use the build script: 
+   https://github.com/apache/teaclave-trustzone-sdk/blob/main/scripts/release/build_dev_docker.sh
+2. Publish the new image to Docker Hub: https://hub.docker.com/u/teaclave 
+   This operation requires access to the Teaclave Docker Hub organization. Please 
+   contact project maintainers for assistance.
+
+### 5. Update CI Configuration for Development Docker
+
+Following the Docker image publication, update the CI workflow to reference the new 
+OP-TEE version.
+
+**Configuration Location:** 
+https://github.com/apache/teaclave-trustzone-sdk/blob/main/.github/workflows/ci.yml#L72
+
+**Affected Jobs:**
+- `test-nostd-build-on-dev-docker`
+- `test-std-build-on-dev-docker`
```

Comment:
@m4sterchain I suggest changing the docker image name from `teaclave-trustzone-emulator-std-optee-4.7.0-expand-memory:latest` to `teaclave-trustzone-emulator-std-expand-memory:optee-4.7.0`. Then CI can always use the tag `teaclave-trustzone-emulator-std-expand-memory:latest`, no need to change the version number for each OP-TEE release.
If it's acceptable I will open a followup PR for this.